data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/source/KafkaSource.java

```diff
@@ -41,95 +53,199 @@
 @SuppressWarnings("deprecation")
 @DataPrepperPlugin(name = "kafka", pluginType = Source.class, pluginConfigurationType = KafkaSourceConfig.class)
 public class KafkaSource implements Source<Record<Object>> {
-  private static final Logger LOG = LoggerFactory.getLogger(KafkaSource.class);
-  private final KafkaSourceConfig sourceConfig;
-  private ExecutorService executorService;
-  private static final String KAFKA_WORKER_THREAD_PROCESSING_ERRORS = "kafkaWorkerThreadProcessingErrors";
-  private final Counter kafkaWorkerThreadProcessingErrors;
-  private final PluginMetrics pluginMetrics;
-  private String consumerGroupID;
-  private String pipelineName;
-  private static String schemaType = "";
-  private MultithreadedConsumer multithreadedConsumer;
-  private int totalWorkers;
-
-  @DataPrepperPluginConstructor
-  public KafkaSource(final KafkaSourceConfig sourceConfig, final PluginMetrics pluginMetrics,
-                     final PipelineDescription pipelineDescription) {
-    this.sourceConfig = sourceConfig;
-    this.pluginMetrics = pluginMetrics;
-    this.pipelineName = pipelineDescription.getPipelineName();
-    this.kafkaWorkerThreadProcessingErrors = pluginMetrics.counter(KAFKA_WORKER_THREAD_PROCESSING_ERRORS);
-  }
+    private static final Logger LOG = LoggerFactory.getLogger(KafkaSource.class);
+    private final KafkaSourceConfig sourceConfig;
+    private ExecutorService executorService;
+    private static final String KAFKA_WORKER_THREAD_PROCESSING_ERRORS = "kafkaWorkerThreadProcessingErrors";
+    private final Counter kafkaWorkerThreadProcessingErrors;
+    private final PluginMetrics pluginMetrics;
+    private String consumerGroupID;
+    private MultithreadedConsumer multithreadedConsumer;
+    private int totalWorkers;
+    private String pipelineName;
+    private static String schemaType = "";
```

Comment:
This should be set to `MessageFormat.PLAINTEXT.toString()` instead of setting it in line 144.